io.github.Fndroid.clash_for_windows.yaml

```diff
@@ -0,0 +1,57 @@
+app-id: io.github.Fndroid.clash_for_windows
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+base: org.electronjs.Electron2.BaseApp
+base-version: 21.08
+sdk: org.freedesktop.Sdk
+separate-locales: false
+command: cfw.sh
+
+finish-args:
+- --share=ipc
+- --share=network
+- --socket=x11
+- --device=dri
+- --filesystem=xdg-config/clash:create
+- --filesystem=xdg-config/autostart:create
```

Comment:
Autostart entries it'll create won't work anyway, so this permission is an unnecessary sandbox escape.

Reply:
> Autostart entries it'll create won't work anyway, so this permission is an unnecessary sandbox escape.

Thanks, after testing, it does not work, I will fix it right away.